uniffi/src/bindings/swift/gen_swift.rs

```diff
@@ -76,96 +76,72 @@ mod filters {
     use super::*;
     use std::fmt;
 
-    /// Declares a C type in the bridging header.
-    pub fn decl_c(type_: &TypeReference) -> Result<String, askama::Error> {
+    /// Declares a Swift type in the public interface for the library.
+    pub fn type_swift(type_: &Type) -> Result<String, askama::Error> {
```

Comment:
These were renamed and re-ordered for consistency with the other language backends.

Reply:
I've been meaning to do this. Thank you.